Common/src/config_structure.cpp

```diff
@@ -1492,6 +1492,17 @@ void CConfig::SetConfig_Options() {
   /*!\brief CONV_FIELD\n DESCRIPTION: Output field to monitor \n Default: depends on solver \ingroup Config*/
   addStringListOption("CONV_FIELD", nConvField, ConvField);
 
+  /*!\brief WND_CONV_STARTITER\n DESCRIPTION: Iteration number after START_ITER_WND  to begin convergence monitoring\n DEFAULT: 15 \ingroup Config*/
+  addUnsignedLongOption("WND_CONV_STARTITER", Wnd_StartConv_Iter, 15);
+  /*!\brief WND_CONV_CAUCHY_ELEMS\n DESCRIPTION: Number of elements to apply the criteria. \n DEFAULT 100 \ingroup Config*/
+  addUnsignedShortOption("WND_CONV_CAUCHY_ELEMS", Wnd_Cauchy_Elems, 100);
+  /*!\brief WND_CONV_CAUCHY_EPS\n DESCRIPTION: Epsilon to control the series convergence \n DEFAULT: 1e-3 \ingroup Config*/
+  addDoubleOption("WND_CONV_CAUCHY_EPS", Wnd_Cauchy_Eps, 1E-3);
+  /*!\brief WND_CAUCHY_CRIT\n DESCRIPTION: Determines, if the cauchy convergence criterion should be used for windowed time averaged objective functions*/
+  addBoolOption("WND_CAUCHY_CRIT",Wnd_Cauchy_Crit, false);
+   /*!\brief WND_CONV_FIELD
+    * \n DESCRIPTION: Output fields  for the Cauchy criterium for the TIME iteration. The criterium is applied to the windowed time average of the chosen funcion. */
+  addStringListOption("WND_CONV_FIELD",nWndConvField, WndConvField);
```

Comment:
Here on the "user side" I think options should be as descriptive as possible, to that end WND -> WINDOW.
Also @talbring and @rsanfer prefixed all convergence related options with CONV_ I think we should stick to that convention, please move WINDOW to after CONV or to the end of the option.
As a general question, can the windowing feature be used as a convergence criteria for time domain while the normal Cauchy criteria is used for the inner iterations? (trying to understand if we really need these new options or if the existing ones could be re purposed.)

Reply:
> As a general question, can the windowing feature be used as a convergence criteria for time domain while the normal Cauchy criteria is used for the inner iterations? (trying to understand if we really need these new options or if the existing ones could be re purposed.)

Yes the windowing feature can be used as a convergence criterion for the time domain independently of the normal Cauchy criterion. You can use both at the same time, just one or none at all. Therefore I think it is useful to define a extra set  options for the windowed time convergence criterion.
In my experience, the windowed-time-cauchy-convergence-criterion needs different parameters than the normal one. For example in a test case, which exhibits only approximately periodic behaviour, one can use the windowed-time-cauchy-convergence-criterion, but usually, the convergence rate is quite slow and it may be usefull to define an EPSILON value, which is much bigger than a reasonable EPSILON for the inner convergence.